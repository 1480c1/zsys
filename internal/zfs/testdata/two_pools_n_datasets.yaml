pools:
  - name: rpool
    datasets:
      - name: ROOT
      - name: ROOT/ubuntu
        zsys_bootfs: yes
        last_used: 2019-04-18T02:45:55+00:00
        mountpoint: /
        canmount: on
  - name: bpool
    datasets:
      - name: BOOT
      - name: BOOT/boot
        zsys_bootfs: yes
        last_used: 2019-04-18T02:45:55+00:00
        mountpoint: /boot
        canmount: on

