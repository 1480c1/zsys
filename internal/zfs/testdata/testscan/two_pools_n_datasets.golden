[
   {
      "Name": "bpool",
      "IsSnapshot": false,
      "Mountpoint": "/",
      "CanMount": "off",
      "BootFS": "",
      "LastUsed": 0,
      "SystemDataset": "",
      "Origin": ""
   },
   {
      "Name": "bpool/BOOT",
      "IsSnapshot": false,
      "Mountpoint": "/BOOT",
      "CanMount": "off",
      "BootFS": "",
      "LastUsed": 0,
      "SystemDataset": "",
      "Origin": ""
   },
   {
      "Name": "bpool/BOOT/boot",
      "IsSnapshot": false,
      "Mountpoint": "/boot",
      "CanMount": "on",
      "BootFS": "yes",
      "LastUsed": 1555555555,
      "SystemDataset": "",
      "Origin": ""
   },
   {
      "Name": "rpool",
      "IsSnapshot": false,
      "Mountpoint": "/",
      "CanMount": "off",
      "BootFS": "",
      "LastUsed": 0,
      "SystemDataset": "",
      "Origin": ""
   },
   {
      "Name": "rpool/ROOT",
      "IsSnapshot": false,
      "Mountpoint": "/ROOT",
      "CanMount": "off",
      "BootFS": "",
      "LastUsed": 0,
      "SystemDataset": "",
      "Origin": ""
   },
   {
      "Name": "rpool/ROOT/ubuntu",
      "IsSnapshot": false,
      "Mountpoint": "/",
      "CanMount": "on",
      "BootFS": "yes",
      "LastUsed": 1555555555,
      "SystemDataset": "",
      "Origin": ""
   }
]